### ISSUE TITLE
Does the test suite require bats-support to be installed via nodemodules ?

### ISSUE THREAD

**Issue by MichaelDimmitt** [bats-core/bats-assert#38]
bats-assert/test/test_helper.bash
```bash
# Load dependencies.
load "${BATS_TEST_DIRNAME}/../node_modules/bats-support/load.bash"

# Load library.
load '../load'
```

**Comment by martin-schulze-vireso** [comment]
See also #34

**Comment by martin-schulze-vireso** [comment]
Fixed via #49
